config/jobs/kubernetes/sig-testing/conformance-e2e.yaml

```diff
@@ -32,7 +32,7 @@ presubmits:
             - "bash"
             - "--"
             - "-c"
-            - "cd ./../../k8s.io/kubernetes && source ./../test-infra/experiment/kind-conformance-e2e.sh"
+            - "cd ./../../k8s.io/kubernetes && source ./../sigs.k8s.io/kind/hack/e2e.sh"
```

Comment:
This is the line that 'hack/update-config.sh' is failing on.

```
INFO: Running command line: bazel-bin/config/jobs/kubernetes-security/linux_amd64_stripped/genjobs '--config=/home/hippie/go/src/k8s.io/test-infra/prow/config.yaml' '--jobs=/home/hippie/go/src/k8s.io/test-infra/config/jINFO: Build completed successfully, 1 total action
3131,3132c3131
<         - cd ./../../k8s.io/kubernetes && source ./../../sigs.k8s.io/kind/hack/e2e.sh
<           && echo ? #added temporary echo to ensure it was this line
---
>         - cd ./../../k8s.io/kubernetes && source ./../test-infra/experiment/kind-conformance-e2e.sh

FAILED: config is not correct, please run 'hack/update-config.sh'
```

Reply:
kubernetes-security is auto generated by hack/update-config.sh